mypy/test/data.py

```diff
@@ -65,6 +65,8 @@ def parse_test_cases(
             tcout = []  # type: List[str]
             if i < len(p) and p[i].id == 'out':
                 tcout = p[i].data
+                if p[i].arg == 'pathfix':
+                    tcout = [s.replace('/', os.path.sep) for s in tcout]
```

Comment:
I considered this but was hesitant to add more magic to this code. If that's the preference, I certainly could, though. Do you suppose there are other [out] sections that might be affected by that change? Should that behavior be documented somewhere, maybe in comments in test/data.py?